lib/phoenix_live_view/html_engine.ex

```diff
@@ -648,20 +650,23 @@ defmodule Phoenix.LiveView.HTMLEngine do
     {let, [quoted_value | r], a}
   end
 
+  # TODO: deprecate "let" in favor of `:let`
   defp split_component_attr(
-         {"let", {:expr, value, %{line: line, column: col} = meta}},
+         {let, {:expr, value, %{line: line, column: col} = meta}},
          {nil, r, a},
          file
-       ) do
+       )
+       when let in ["let", ":let"] do
     quoted_value = Code.string_to_quoted!(value, line: line, column: col, file: file)
     {{quoted_value, meta}, r, a}
   end
 
   defp split_component_attr(
-         {"let", {:expr, _value, previous_meta}},
+         {let, {:expr, _value, previous_meta}},
          {{_, meta}, _, _},
          file
-       ) do
+       )
+       when let in ["let", ":let"] do
     message = """
     cannot define multiple `let` attributes. \
     Another `let` has already been defined at line #{previous_meta.line}\
```

Comment:
In error messages and docs, let's move to `:let`:

```suggestion
    cannot define multiple `:let` attributes. \
    Another `:let` has already been defined at line #{previous_meta.line}\
```